include/mp/proxy-io.h

```diff
@@ -66,8 +66,6 @@ struct ProxyClient<Thread> : public ProxyClientBase<Thread, ::capnp::Void>
     ProxyClient(const ProxyClient&) = delete;
     ~ProxyClient();
 
-    void setDisconnectCallback(const std::function<void()>& fn);
```

Comment:
In 7a8a99c50feca10be4e42a1cdb135864ce376f71 _bug: fix ProxyClient<Thread> deadlock if disconnected as IPC call is returning_: can you dedicate a sentence in the commit message as to why this function was dropped?

Reply:
I agree it's nicer to drop it.